Formula/tsduck.rb

```diff
@@ -0,0 +1,25 @@
+class Tsduck < Formula
+  desc "MPEG Transport Stream Toolkit"
+  homepage "https://tsduck.io/"
+  url "https://github.com/tsduck/tsduck/archive/v3.28-2551.tar.gz"
+  sha256 "ff39c9b97e17a5ae06556a119e24783427a86b1bb5ac75c7ba0340c7bd07d02d"
+  license "BSD-2-Clause"
+  head "https://github.com/tsduck/tsduck.git", branch: "master"
+
+  depends_on "dos2unix" => :build
+  depends_on "gnu-sed" => :build
+  depends_on "grep" => :build
+  depends_on "openjdk" => :build
+  depends_on "librist"
+  depends_on "pcsc-lite"
+  depends_on "srt"
+
+  def install
+    system "make", "NOGITHUB=1", "NOTEST=1", "-j8"
```

Comment:
OK, but are you sure it is optimum? The project is quite big (more than 800 C++ source files) and compiling it with the default takes ages. Users who build from head complain about this. On my quad-core i7 iMac, I use "make -j12", more than the default, but the only acceptable solution on a user experience standpoint. How is it possible to specify "more than default"?

Reply:
By setting `HOMEBREW_MAKE_JOBS` env var IIRC. Users in most cases install binary packages rather than compiling from source, I wouldn't worry about it.